scripts/sysman/common.yml

```diff
@@ -211,6 +211,10 @@ etors:
     - name: DEVICE_UUID
       desc: $s_uuid_t
       version: "1.7"
+    - name: MEMORY_EXP_STATE
```

Comment:
should be `MEMORY_STATE_EXP`? @wdamon-intel any guidance on this?

Reply:
@aravindksg `MEMORY_STATE_EXP` is correct

Reply:
Fixed in common.yml